tests/support/mod.rs

```diff
@@ -214,32 +254,20 @@ impl RedisServer {
                     .arg(&path);
                 RedisServer {
                     process: spawner(&mut redis_cmd),
-                    stunnel_process: None,
                     tempdir: Some(tempdir),
                     addr,
                 }
             }
         }
     }
 
-    pub fn wait(&mut self) {
```

Comment:
I couldn't find anything that used this, even with different sets of features turned on, so I removed it.